reddit_gold/__init__.py

```diff
@@ -28,6 +28,7 @@ class Gold(Plugin):
         ),
 
         "snoovatar": LocalizedModule("snoovatar.js",
+            "paper-core.min.js",
```

Comment:
ah, I was only looking at the first commit when I made the comment about `svg.min.js`, but it still applies here